src/main/scala/org/broadinstitute/dsde/firecloud/webservice/SnapshotApiService.scala

```diff
@@ -20,6 +21,13 @@ trait SnapshotApiService extends FireCloudDirectives with UserInfoDirectives {
         pathEnd {
           post {
             passthrough(SnapshotApiService.createDataRepoSnapshotURL(namespace, name), HttpMethods.POST)
+          } ~
+          get {
+            pathPrefix("workspaces" / Segment / Segment / "snapshots") { (namespace, name) =>
```

Comment:
sorry, one more tweak - need to remove the `pathPrefix("workspaces" / Segment / Segment / "snapshots")` here. This is already inside a `pathPrefix("workspaces" / Segment / Segment / "snapshots")` at line 20. All the nested path directives (including the `pathEnd` at line 21) combine to define the path - this extra line makes it invalid!

Reply:
good catch!